prow/plugins.yaml

```diff
@@ -27,6 +27,36 @@ label:
   # responsible for maintaining the milestones:
   # curl -H "Authorization: token <token>" https://api.github.com/orgs/<org-name>/teams
   milestone_maintainers_id: 2460384
+  slacksigs:
+  - sig-api-machinery
```

Comment:
Btw, this list should include only sig-testing for start. Make an announcement in kubernetes-dev and any interested SIGs should be able to open PRs and add themselves.

Reply:
I don't think that is how people operate...

There are huge outcome differences depending on whether you ask new employees to opt-in or opt-out of retirement planning like a 401K. 

Maybe limit it to sig-testing until we have more confidence the plugin is good and then foist it upon everyone else?